general/g.mapset/tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+from types import SimpleNamespace
+
+import grass.script as gs
+import pytest
+
+TEST_MAPSETS = ["PERMANENT", "test1", "test2", "test3"]
+
+
+@pytest.fixture(scope="module")
+def simple_dataset(tmp_path_factory):
+    """Set up a GRASS session for the tests."""
+    tmp_path = tmp_path_factory.mktemp("simple_dataset")
+    location = "test"
```

Comment:
For new code, let's use project instead of old name location.
```suggestion
    project = "test"
```